e2e/__snapshots__/fablo-config-hlf1.3-2orgs-1chaincode-private-data.json.test.ts.snap

```diff
@@ -3177,12 +3202,9 @@ chaincodeBuild() {
       fi
     fi
 
-    NODE_VERSION=\\"$(node --version)\\"
-
-    USES_OLD_FABRIC_SHIM=\\"$(jq '.dependencies.\\"fabric-shim\\" | contains(\\"1.4.\\")' \\"$CHAINCODE_DIR_PATH/package.json\\")\\"
-    if [ \\"$USES_OLD_FABRIC_SHIM\\" == \\"true\\" ]; then
-      RECOMMENDED_NODE_VERSION=\\"8.9\\"
-    fi
+    NODE_VERSION=$(node --version)
+    fabric_shim_version=$(jq -r '.dependencies.\\"fabric-shim\\"' \\"$CHAINCODE_DIR_PATH/node_modules/fabric-shim/package.json\\")
```

Comment:
It seems changing the path broke tests and making it work with checking fabric-shim inside node_modules seems a bit complex. My suggestion is to bring back the old file (`$CHAINCODE_DIR_PATH/package.json`), and create an issue for changing it to `$CHAINCODE_DIR_PATH/node_modules/fabric-shim/package.json`. It can be resolved later. Could you revert it, fix tests, and create an issue for it?

Reply:
yeah sure so right now its fine if I check for version in `$CHAINCODE_DIR_PATH/package.json` right?

Reply:
yes, exactly